### DISCUSSION TITLE
Welcome to position-protocol Discussions!

### DISCUSSION THREAD

**Discussion by JustInPosition** [PositionExchange/position-protocol#14] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PedroArvina** [comment]
Good night, can anyone tell me how the position has the pankake exange without making a closed fork? how can we buy on pankake ofc and platform?

**Reply by vtminhvt** [reply]
Hi all
